CMakeLists.txt

```
@@ -179,9 +185,18 @@ if("${BUILD_TESTING}")
         DEPENDS ${PROJECT_NAME}
     )
     nwx_pybind11_tests(
-        py_pluginplay ${python_tests_dir}/unit_tests/test_pluginplay.py
+        py_pluginplay ${python_test_dir}/unit_tests/test_pluginplay.py
         SUBMODULES parallelzone
     )
+    nwx_add_pybind11_module(
+        ${PROJECT_NAME}_examples
+        SOURCE_DIR "${python_test_dir}/doc_snippets"
+        DEPENDS parallelzone ${PROJECT_NAME} ${PROJECT_NAME}_examples
+    )
+    nwx_pybind11_tests(
+        py_doc_snippets "${python_test_dir}/doc_snippets/test_doc_snippets.py"
```

```suggestion
        py_${PROJECT_NAME}_docs "${python_test_dir}/doc_snippets/test_doc_snippets.py"
```

The cxx tests are `test_pluginplay` and `test_pluginplay_docs`, so if the python unit tests are `py_pluginplay` (corresponding to `test_pluginplay`), then the python doc snippet tests could be `py_pluginplay_docs`.